components/LogBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from "next/link";
+
+interface ILogBannerProps {
+  title: string;
+  content: string;
+  timePublished: string;
+  path: string;
+}
+
+export default function LogBanner({
+  title,
+  content,
+  timePublished,
+  path,
+}: ILogBannerProps) {
+  return (
+    <div className="pt-10">
+      <Link href={path}>
+        <div className="h-[225px] w-[930px] bg-light-gray px-8 pt-6">
+          <h2 className="text-3xl font-semibold">
+            {title + " - "}
+            <time>{timePublished}</time>
```

Comment:
add the datetime attribute to tag time tag, like: `datetime="2008-02-14 20:00"`

Reply:
I'm not sure if what I did was the desired method, but it's done